docs/template_plugin/src/template_plugin.cpp

```diff
@@ -152,14 +152,26 @@ InferenceEngine::QueryNetworkResult Plugin::QueryNetwork(const InferenceEngine::
     // So we need store as supported either unsupported node sets
     std::unordered_set<std::string> supported;
     std::unordered_set<std::string> unsupported;
-    auto opset = ngraph::get_opset4();
-    for (auto&& node : transformedFunction->get_ops()) {
+    std::vector<ngraph::OpSet> opsets{ ngraph::get_opset1(),
+                                       ngraph::get_opset2(),
+                                       ngraph::get_opset3(),
+                                       ngraph::get_opset4(),
+                                       ngraph::get_opset5(),
+                                       ngraph::get_opset6(),
+                                       ngraph::get_opset7(),
+                                       ngraph::get_opset8() };
+    std::set<ngraph::NodeTypeInfo> op_super_set;
```

Comment:
why not `ngraph::OpSet` instead of `std::set<ngraph::NodeTypeInfo>`?

Reply:
Fixed, please review again)